homeassistant/components/hue/sensor_base.py

```diff
@@ -280,5 +280,7 @@ class GenericZLLSensor(GenericHueSensor):
     def device_state_attributes(self):
         """Return the device state attributes."""
         return {
-            "battery_level": self.sensor.battery
+            "battery_level": self.sensor.battery,
+            "last_updated": self.sensor.state['lastupdated'],
```

Comment:
We shouldn't include this. Each entity state in Home Assistant already has a last_changed and last_updated.